index.css

```diff
@@ -1,68 +1,58 @@
 @import "https://fonts.googleapis.com/css?family=Lato:400,700";
 
+:root {
+  --body-color: white;
+  --main-color: #f5f5f0;
+  --head-color: #c2d6c1;
+  --font-color: #29a329;
+}
+
 * {
   box-sizing: border-box;
 }
 
 body {
-  background-color: white;
+  background: var(--body-color);
   margin: 0;
   font-family: "Lato", sans-serif;
 }
 
 .container {
   display: flex;
-  margin: 4rem;
-}
-
-.products-section {
-  background: #f5f5f0;
-  width: 30%;
-}
-
-.products-section__title {
-  width: 100%;
-  display: flex;
-  justify-content: center;
-  background: #b0e5ad;
-  color: #29a329;
-  font-size: 1.3rem;
-  font-weight: bold;
-  padding: .5rem;
+  padding: 4rem;
```

Comment:
Po zobaczeniu tego "na żywo", uznałem, że lepiej będzie jak wszystkie dystanse pomiędzy boxami i krawędziami ekranu zmniejszymy do 2.5rem. Za dużo przestrzeni tracimy w tym momencie.

I te 2.5rem też bym wyrzucił do zmiennej i użył we wszystkich 3 miejscach, które jej dotyczą.